### PR TITLE
[new release] utop (2.2.0)

### PR DESCRIPTION
**Pull request by rgrinberg**
CHANGES:

* 4.07.0 compatibility (diml/utop#238, @hcarty)
* Fix compatibility with latest tuareg-mode (diml/utop#241, @Wilfred)
* Do not expand include directories (diml/utop#242, @sliquister)

### CODE REVIEW

File: packages/utop/utop.2.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "jeremie@dimino.org"
+authors: ["Jérémie Dimino"]
+license: "BSD3"
+homepage: "https://github.com/diml/utop"
+bug-reports: "https://github.com/diml/utop/issues"
+dev-repo: "https://github.com/diml/utop.git"
+build: [
+  ["jbuilder" "subst"] {pinned}
```
Comment (kit-ty-kate) [inline packages/utop/utop.2.2.0/opam]: Could you add `"-p" name` here? See https://github.com/ocaml/opam-repository/pull/12279

**Comment by kit-ty-kate** [comment]
Otherwise everything seems good.
There is only one unrelated failure: `http://gallium.inria.fr/~fpottier/menhir/menhir-20171013.tar.gz` doesn't seem to be accessible anymore. cc @fpottier

**Comment by kit-ty-kate** [comment]
Mmh something very strange is happening :/
```
+wget http://caml.inria.fr/pub/distrib/ocaml-4.06/ocaml-4.06.1.tar.gz
--2018-07-15 12:09:59--  http://caml.inria.fr/pub/distrib/ocaml-4.06/ocaml-4.06.1.tar.gz
Resolving caml.inria.fr (caml.inria.fr)... 128.93.101.34
Connecting to caml.inria.fr (caml.inria.fr)|128.93.101.34|:80... connected.
HTTP request sent, awaiting response... 403 Forbidden
2018-07-15 12:10:00 ERROR 403: Forbidden.
```
I'll merge this because it's unrelated anyway. Thanks!

**Comment by kit-ty-kate** [comment]
cc @damiendoligez for the error above

**Comment by rgrinberg** [comment]
Thanks. My internet connectivity is a bit limited, so I couldn't make the fixes on time. Will fix this issue in master as well.

**Comment by damiendoligez** [comment]
Our server rebooted in a strange state after a system upgrade. This is now fixed (by @xavierleroy).

**Pull request merged**